src/mono/wasm/debugger/DebuggerTestSuite/EvaluateOnCallFrameTests.cs

```diff
@@ -1181,8 +1181,7 @@ async Task EvaluateOnCallFrameFail(string call_frame_id, params (string expressi
                 var (refList, _) = await EvaluateOnCallFrame(id, "testPropertiesNone.list");
                 var refListProp = await GetProperties(refList["objectId"]?.Value<string>());
                 var list = refListProp
-                    .Where(v => v["name"]?.Value<string>() == "Items" || v["name"]?.Value<string>() == "_items")
-                    .FirstOrDefault();
+                    .FirstOrDefault(v => v["name"]?.Value<string>() == "Items" || v["name"]?.Value<string>() == "_items");
```

Comment:
You can use `.First` here, so it will throw.
```suggestion
                    .First(v => v["name"]?.Value<string>() is "Items" or "_items");
```